Sources/Danger/DangerDSL.swift

```diff
@@ -16,7 +16,6 @@ public struct DangerDSL: Decodable {
     public private(set) var github: GitHub!
 
     public let bitbucketServer: BitBucketServer!
- mutating
     
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/DangerDSL.swift;
  Line: 19;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)